qemu/tests/commit_snapshot_to_raw_backing.py

```diff
@@ -80,7 +81,21 @@ def _verify_qemu_img_info(output, b_fmt, b_name):
                      snapshot, base, cache_mode)
     else:
         logging.info("Commit snapshot image %s back to %s.", snapshot, base)
+
+    org_size = json.loads(sn_img.info(output="json"))["actual-size"]
     sn_img.commit(cache_mode=cache_mode)
+    remain_size = json.loads(sn_img.info(output="json"))["actual-size"]
+
+    """Verify the snapshot file whether emptied after committing"""
+    logging.info("Verify the snapshot file whether emptied after committing")
+    commit_size = org_size - remain_size
+    dd_bs = int(params["dd_bs_val"])
+    dd_count = int(params["dd_bs_count"])
+    dd_size = eval('dd_bs * dd_count * 1024 * 1024')
```

Comment:
For line 92~94, you could do like this:
`dd_size = eval('params["dd_bs_val"] * params["dd_bs_count"] * 1024 ** 2')`